src/west/manifest.py

```diff
@@ -211,7 +211,8 @@ def _manifest_content_at(project: 'Project', path: PathType,
 _import_map = collections.namedtuple('_import_map',
                                      'file '
                                      'name_whitelist path_whitelist '
-                                     'name_blacklist path_blacklist')
+                                     'name_blacklist path_blacklist '
```

Comment:
Not to be picky but `typing.NamedTuple` is much nicer IMHO

Reply:
> Not to be picky but `typing.NamedTuple` is much nicer IMHO

Neat! I didn't know this existed.